debian/overlay-snapshots.postinst

```diff
@@ -1,17 +0,0 @@
-# overlay-snapshots.postinst
-#
-# Copyright (C) 2018 Kano Computing Ltd.
-# License: http://www.gnu.org/licenses/gpl-2.0.txt GNU GPL v2
-#
-# TODO
-
-
-case "$1" in
-    configure)
-        sed -i 's|$| init=/bin/ovlsnap-init|' /boot/cmdline.txt
```

Comment:
Don't be like the programmer who added this code, be like the one who removed it. 

Say **NO** to `sed`.